### PR TITLE
Edits to reflect new Sedai logo, marketing content and an out of the box dashboard

### PR DESCRIPTION
**Pull request by praveenprakash**
### What does this PR do?

Edits to reflect new Sedai Logo, Marketing message in readme & manifest and an out of the box dashboard.

### Motivation

To make sedai available as an integration tile

### Review checklist

- [ x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: sedai/README.md
Comment (anawishnoff) [inline sedai/README.md:3]: Curious about what the word opportunities is referring to here? Also, we generally try and stay away from including marketing/sales language in free integrations so I'd recommend removing the learn more and try for free sentence.

File: sedai/README.md
Comment (anawishnoff) [inline sedai/README.md:5]:
```suggestion
Enable this integration to receive notifications in Datadog about actions that Sedai autonomously executes in your production environments.
```

File: sedai/README.md
Comment (anawishnoff) [inline sedai/README.md:13]:
```suggestion
* **Proactive Actions:** Safely acts in production on your behalf to ensure that resources avoid availability issues and run optimally at all times.
```

**Comment by anawishnoff** [comment]
Hi @praveenprakash, thank you for opening this pull request! I left a few comments for you on your files and also had a couple of general comments:

- Your dashboard is looking great, the only improvement I would suggest is to add titles to all of the widgets. Some widgets are still displaying their default titles (see screenshot below) and it would be great if everything was consistent with proper titles across the dashboard.
    - Screen Shot 2022-03-28 at 11 27 46 AM
- I see that image files are included with this PR, but the images are not included anywhere in your readme file. To add images to your readme, simply add them in-line with the following markdown syntax:
`    alt text`
- In your manifest.json file, the `public_title` field is what will be displayed as the title on your tile. I'd recommend changing it to "Sedai Integration". 

Let me know if you have any questions. Thanks!

File: sedai/README.md
Comment (praveenprakash) [reply sedai/README.md:24]: Sure thanks..

**Comment by praveenprakash** [comment]
> Hi @praveenprakash, thank you for opening this pull request! I left a few comments for you on your files and also had a couple of general comments:
> 
> * Your dashboard is looking great, the only improvement I would suggest is to add titles to all of the widgets. Some widgets are still displaying their default titles (see screenshot below) and it would be great if everything was consistent with proper titles across the dashboard.
>   
>   * Screen Shot 2022-03-28 at 11 27 46 AM
> * I see that image files are included with this PR, but the images are not included anywhere in your readme file. To add images to your readme, simply add them in-line with the following markdown syntax:
>   `    alt text`
> * In your manifest.json file, the `public_title` field is what will be displayed as the title on your tile. I'd recommend changing it to "Sedai Integration".
> 
> Let me know if you have any questions. Thanks!

Hi @anawishnoff ,
Thanks for the comments.. I'm pretty new to Datadog and so need some clarifications on the comments above, please find them below..

#2. Images - The images included are not used in ReadMe - its a carry over from the previous PR.  But I think its good to use those images in the Readme, is it OK to include them under the setup section..? I couldn't find a documentation on the use of images in readme at this doc so asking.. https://docs.datadoghq.com/developers/marketplace/#readme
If there are any references that I can use, could you please share..?

3. public_title -  I'm referring to the below documentation for manifest.json.. and it states 

> "display_name" will be "title displayed on the integration tile"


Also, it states `"public_title" is the "title of the integration displayed on the documentation. Should follow the following format: <INTEGRATION_NAME>."`
I'm not clear what is meant by `'documentation'` in the above line & with the format specification, does it need to be _**"SEDAI_INTEGRATION"**_..?

https://docs.datadoghq.com/developers/integrations/check_references/?tab=manifestversion1#manifest-file

(By the way, I accidentally clicked "Close with Comment" and that closed the PR - I've reopened it)

Thanks in advance..,
Praveen

**Comment by anawishnoff** [comment]
Hi @praveenprakash, good questions.

For the `public_title`, please disregard my previous comment, I had gotten confused between those two fields. I believe the `public_title` should be "Datadog-Sedai Integration". 

For the images, yes you can add those images directly inline in the Setup section of the readme. For an example, take a look at the [readme from the Pure Storage FlashArray integration](https://github.com/DataDog/integrations-extras/tree/master/purefa#configuration), you can see they've added an image inline to their setup section.

Let me know if anything needs more clarification. Thanks!

**Comment by praveenprakash** [comment]
> Hi @praveenprakash, good questions.
> 
> For the `public_title`, please disregard my previous comment, I had gotten confused between those two fields. I believe the `public_title` should be "Datadog-Sedai Integration".
> 
> For the images, yes you can add those images directly inline in the Setup section of the readme. For an example, take a look at the [readme from the Pure Storage FlashArray integration](https://github.com/DataDog/integrations-extras/tree/master/purefa#configuration), you can see they've added an image inline to their setup section.
> 
> Let me know if anything needs more clarification. Thanks!

Thanks @anawishnoff for the quick response, 
I've passed on the content change suggestions to our team and we are working on updating the content and adding images to the readme content - will update back here when those changes are pushed.

Thanks,
Praveen

**Comment by anawishnoff** [comment]
Hi @praveenprakash, just checking in to see if there's anything we can help with here or if you have any questions. Thanks!

**Comment by praveenprakash** [comment]
> 

Hi @anawishnoff, thanks for checking - no I'm waiting on our content team for the edits.. will update here once I push the edits to the PR.

**Comment by praveenprakash** [comment]
Hi @anawishnoff, 
Just FYI - I've updated the PR with edits for your review comments. Please take a look when you have a chance.
Thanks,
Praveen

File: sedai/README.md
Comment (anawishnoff) [inline sedai/README.md:22]:
```suggestion
* Enter a nickname and the API Key for your Datadog account, enable it, and test the integration.
```

File: sedai/README.md
Comment (anawishnoff) [inline sedai/README.md:19]:
```suggestion
* Navigate to Settings > Notifications > Add Integration > Datadog Icon 
```

File: sedai/README.md
Comment (anawishnoff) [inline sedai/README.md:25]:
```suggestion
* Once the test is verified to be working, click Save.
```

File: sedai/README.md
Comment (anawishnoff) [inline sedai/README.md:33]: Apologies if we've already discussed this, but isn't Sedai sending data into Datadog through events? If not, what kind of data is being sent in?

File: sedai/manifest.json
Comment (anawishnoff) [inline sedai/manifest.json:9]:
```suggestion
  "short_description": "An autonomous platform to intelligently manage your cloud applications",
```

**Review by anawishnoff — CHANGES_REQUESTED** [review]
Thanks for making these updates @praveenprakash! I added a few comments, also please note the validation errors with the way that images are linked in the README file.

File: sedai/README.md
Comment (praveenprakash) [reply sedai/README.md:33]: Hi @anawishnoff,
Yes, you are right - we send in Notifications on Sedai events that the user has enabled in set up steps to Datadog through the Events API.
Do you think the following line would do..? If not, could you please share your suggestion/recommendation..?

"This integration sends notifications on subscribed events from Sedai to Datadog."

**Comment by praveenprakash** [comment]
Hi @anawishnoff ,
Pushed an update to the PR with image path fixes, accepting the text changes except the "Data Collected" section - will update it once I hear from you on my request for recommendation on it.

Appreciate your help with this review.

Regards,
Praveen

File: sedai/README.md
Comment (anawishnoff) [inline sedai/README.md:11]:
```suggestion
* **Configuration-free:** Easily connects to Datadog API and intelligently identifies, prioritizes and learns metric behavior.
```

File: sedai/assets/dashboards/sedai_overview.json
Comment (anawishnoff) [inline sedai/assets/dashboards/sedai_overview.json:27]:
```suggestion
                "title": "Sedai Events",
```

**Review by anawishnoff — APPROVED** [review]
Left two minor comments, otherwise looks good from the product side!

**Comment by praveenprakash** [comment]
Thanks @anawishnoff ,
I've accepted the last 2 suggestions - thanks for those.
Will the PR be merge to master automatically or is there anything left from my side to be done for it to be merged.?

Thanks,
Praveen

File: sedai/README.md
Comment (fanny-jiang) [inline sedai/README.md:20]:
```suggestion

![DataDog Notification Integration][3]
...
[3]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/sedai/images/DataDog_Notification_Integration.png)
```

Insert a newline so that the images aren't inline with the previous line of text. Also, can we move the image URLs to the reference links section?

File: sedai/README.md
Comment (fanny-jiang) [inline sedai/README.md:23]:
```suggestion

![DataDog Notification Integration](https://raw.githubusercontent.com/DataDog/integrations-extras/master/sedai/images/Add_DataDog_Channel.png)
```

File: sedai/README.md
Comment (fanny-jiang) [inline sedai/README.md:26]:
```suggestion

![Working DataDog Integration](https://raw.githubusercontent.com/DataDog/integrations-extras/master/sedai/images/Add_DataDog_Channel-Working_REC.png)
```

File: sedai/README.md
Comment (fanny-jiang) [inline sedai/README.md:29]:
```suggestion

![Enable DataDog Notifications](https://raw.githubusercontent.com/DataDog/integrations-extras/master/sedai/images/Enable_Notifications.png)
```

**Review by fanny-jiang — CHANGES_REQUESTED** [review]

File: sedai/README.md
Comment (praveenprakash) [reply sedai/README.md]: Sure, done and PR is updated.

**Comment by praveenprakash** [comment]
Hi @fanny-jiang ,
The edits are made and the PR is updated.
Thanks,
Praveen

File: sedai/README.md
Comment (praveenprakash) [reply sedai/README.md]: PR Updated with changes for review comments

File: sedai/README.md
Comment (praveenprakash) [reply sedai/README.md]: PR Updated with changes for review comments

File: sedai/README.md
Comment (cswatt) [inline sedai/README.md:23]:
```suggestion
2. Enter a nickname and the API key for your Datadog account. Enable and test the integration.
```

File: sedai/README.md
Comment (cswatt) [inline sedai/README.md:41]:
```suggestion
For help with this integration, contact [Datadog support][1].
```

File: sedai/README.md
Comment (cswatt) [inline sedai/README.md:44]:
```suggestion
[1]: https://docs.datadoghq.com/help/
```

File: sedai/assets/dashboards/sedai_overview.json
Comment (cswatt) [inline sedai/assets/dashboards/sedai_overview.json:3]:
```suggestion
    "description": "Sedai is an autonomous, agentless, and configuration-free SaaS that intelligently manages your cloud environments to prevent availability issues and optimize performance and cost—all in production. Sedai's Continuous Autonomous Availability Management (CAAM) platform helps you take full advantage of your microservices with smart signals that prevent downtime, avoid potential SLO breaches, improve latency, analyze code releases, and lower cloud costs. \n\nSedai User Guide: https://sedai.gitbook.io/sedai/",
```

**Review by cswatt — CHANGES_REQUESTED** [review]

File: sedai/manifest.json
Comment (yzhan289) [inline sedai/manifest.json:3]: In the `CODEOWNERS` [file](https://github.com/DataDog/integrations-extras/blob/master/.github/CODEOWNERS#L102), can you update the maintainer information as well?

File: sedai/README.md
Comment (praveenprakash) [reply sedai/README.md:41]: Hi @cswatt ,
Can you please confirm for support email, I should use Datadog's support email
 I've referred to other Readme(s) in the repo (Purefa, Postman  for example) and the support section shows the integration providers email, not that of Datadog.

File: sedai/assets/dashboards/sedai_overview.json
Comment (praveenprakash) [reply sedai/assets/dashboards/sedai_overview.json:3]: @cswatt, 
could you please review the removal of space before and after the '-' in the line 'and cost - all in..' ..?
it's not intended be read as a single word - but to emphasize with a pause.

File: sedai/manifest.json
Comment (praveenprakash) [reply sedai/manifest.json:3]: @yzhan289 ,
Done. Thanks

File: sedai/assets/dashboards/sedai_overview.json
Comment (praveenprakash) [reply sedai/assets/dashboards/sedai_overview.json]: @cswatt,
I have updated the content after reviewing with our content team - the Em dash is replaced with a comma.

**Review by yzhan289 — APPROVED** [review]
LGTM! Once the logo has been updated internally I can merge and deploy this to prod.

**Comment by praveenprakash** [comment]
> LGTM! Once the logo has been updated internally I can merge and deploy this to prod.

Thanks @yzhan289, I guess no more actions are required from my side.

**Pull request merged**
